kani-dependencies

```diff
@@ -1,4 +1,10 @@
-CBMC_VERSION="5.93.0"
+CBMC_MAJOR="5"
+CBMC_MINOR="94"
+CBMC_VERSION="5.94.0"
```

Comment:
Can we set this to:
```
CBMC_VERSION="${CBMC_MAJOR}.${CBMC_MINOR}.0"
```
or add `CBMC_PATCH` as well:
```
CBMC_VERSION="${CBMC_MAJOR}.${CBMC_MINOR}.${CBMC_PATCH}"
```
?

Reply:
I thought about it, but this file is not necessarily a bash script. It can be imported by python scripts as well